internal/convert.go

```diff
@@ -129,6 +130,32 @@ func (c *Converter) addDates(note *enex.Note, md *markdown.Note) {
 	md.MTime = convertEvernoteDate(note.Updated)
 }
 
+const dateFrontMatterFormat = "2006-01-02 15:04:05 -0700"
+
+func (c *Converter) addFrontMatter(note *enex.Note, md *markdown.Note) {
+	var frontMatter = fmt.Sprintf("date: %s\nupdated_at: %s\ntitle: %s\ntags: %s\n",
```

Comment:
I wonder if leveraging a go text template for this block would make the logic look nicer. 
Just thinking out loud. I'm okay with this approach using `fmt.Sprintf` to format the frontmatter line by line.

Reply:
I will change it.